api.py

```diff
@@ -61,6 +61,16 @@ def get_outputs(task_name: str) -> Response:
     return _ask_for(task_name, "/schema/outputs/")
 
 
+@app.route('/schema/outputs/<string:task_name>/<string:aspect_name>')
+def get_output_by_aspect(task_name: str, aspect_name: str) -> Response:
+    """Get specific aspect's output query pattern from algorithm backend
+
+    Returns:
+        Normalized output query pattern definition.
+    """
+    return _ask_for(task_name, "/schema/outputs/{aspect_name}".format(aspect_name=aspect_name))
```

Comment:
This way format will be reversed: to know, how to get to `/schema/outputs/aspect_name/task_name/` we will have to ask for `/schema/outputs/task_name/`. Can we make this straight?

Reply:
Refactored the entire proxying logic to fix the problem you described. Please check commit 9a47973.

Reply:
We are still missing task name in the endpoint url.